spec/03-plugins/26-prometheus/02-access_spec.lua

```diff
@@ -478,11 +479,10 @@ describe("Plugin: prometheus (access) per-consumer metrics", function()
         path    = "/metrics",
       })
       body = assert.res_status(200, res)
-      return body:find('kong_http_status{service="mock-service",route="http-route",code="200"} 1', nil, true)
+      return body:find('kong_request_count{service="mock-service",route="http-route",code="200",source="upstream",consumer="alice"} 1', nil, true)
     end)
 
-    assert.not_match('kong_http_consumer_status{service="mock-service",route="http-route",code="401",consumer="alice"} 1', body, nil, true)
-    assert.matches('kong_http_status{service="mock-service",route="http-route",code="401"} 1', body, nil, true)
+    assert.matches('kong_request_count{service="mock-service",route="http-route",code="401",source="kong",consumer=""} 1', body, nil, true)
 
     assert.matches('kong_nginx_metric_errors_total 0', body, nil, true)
   end)
```

Comment:
Can a test assert that kong_latency, request_latency, upstream_latency time series are present in the output?

Reply:
Added :+1: